--- WSoundPrefs-1.1.1/src/WSoundPrefs.h~	Fri Jan 18 09:47:14 2002
+++ WSoundPrefs-1.1.1/src/WSoundPrefs.h	Fri Jan 18 09:47:14 2002
@@ -42,8 +42,8 @@
 
 #include <wraster.h>
 
-#include <WINGs.h>
-#include <WUtil.h>
+#include <WINGs/WINGs.h>
+#include <WINGs/WUtil.h>
 
 #include <wsound.h>
 
--- WSoundPrefs-1.1.1/src/SystemInfo.c~	Fri Jan 18 09:48:03 2002
+++ WSoundPrefs-1.1.1/src/SystemInfo.c	Fri Jan 18 09:48:03 2002
@@ -32,8 +32,8 @@
 #include <sys/types.h>
 #include <sys/utsname.h>
 #include <pwd.h>
-#include <WINGs.h>
-#include <WUtil.h>
+#include <WINGs/WINGs.h>
+#include <WINGs/WUtil.h>
 
 #include "WSoundPrefs.h"
 #include "SystemInfo.h"
